packages/@ama-sdk/generator-sdk/src/generators/shell/templates/base/package.json

```diff
@@ -68,50 +68,50 @@
     }
   },
   "devDependencies": {
-    "@angular-devkit/core": "~15.1.3",
-    "@angular-devkit/schematics": "~15.1.3",
+    "@angular-devkit/core": "~15.2.0",
+    "@angular-devkit/schematics": "~15.2.0",
     "@angular-devkit/schematics-cli": "~15.1.3",
     "@angular-eslint/eslint-plugin": "~15.2.0",
     "@angular-eslint/eslint-plugin-template": "~15.2.0",
     "@angular-eslint/utils": "~15.2.0",
-    "@angular/compiler": "~15.1.3",
-    "@commitlint/cli": "~16.3.0",
-    "@schematics/angular": "~15.1.3",
-    "@commitlint/config-conventional": "~16.2.0",
+    "@angular/compiler": "~15.2.0",
+    "@commitlint/cli": "^11.0.0",
+    "@schematics/angular": "~15.2.0",
+    "@commitlint/config-conventional": "^11.0.0",
```

Comment:
why are these versions downgraded?

Reply:
Just aligning with the monorepo's versions.
It may mean that the monorepo is not up-to-date